packageScripts/postinstall.mjs

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-const chalk = require('chalk');
+import chalk from 'chalk';
+import { createRequire } from 'node:module';
+const require = createRequire(import.meta.url);
```

Comment:
We should add a note to this file about the total re-write and to see the updated docs.